### PR TITLE
Add support for Partner pages

### PR DESCRIPTION
**Pull request by SantosGuillamot**
We created a Custom Post Type named partner, so we have to support it from Frontity, and we want to start using Contact Form 7, so I installed the package and added some styles.

### CODE REVIEW

**Comment by frontibotito** [comment]
Deploy preview for _website_ ready!

Built with commit cca3e87a8b84797c882155843a55c15b3cff7dc6

https://frontity-org-dyftcb6mt.vercel.app

**Review by michalczaplinski — APPROVED** [review]
LGTM 👍 

I've made a couple of small changes:

- Run `prettier` and `eslint` over the contact-form-7 to enforce the same consistency in style and convention that we have in the rest of the codebase.
- Remove the properties prefixed with `_` from the package.json (they are used by npm internally and show up when you copy the package.json from the `node_modules`)
- Change the `onChange` to `onBlur` in the `<select>` element in `Select.js`

Like mentioned before, I think there is room for improvement in how the contact-form-7 is implemented but we can just merge it as is for now and revisit it in the future to when we decide if we want to make a proper fork or contribute changes upstream.

File: packages/frontity-contact-form-7/package.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
+{
```
[the remark this answers is not in the document]
Comment (michalczaplinski) [reply packages/frontity-contact-form-7/package.json:1]: done 👍

**Comment by SantosGuillamot** [comment]
Great! Thanks Michal :slightly_smiling_face:

**Pull request merged**